### DISCUSSION TITLE
Archive Assets /Inactive

### DISCUSSION THREAD

**Discussion by norameme** [moneymanagerex/moneymanagerex#6336] [Ideas]
Just an idea to make Assets Inactive.
I have an asset that went to the other side after the divorce, and I would still like to keep records connected to it, but it seems I either delete it or have it; there is no other state for the asset.  Maybe we can have an inactive one/ or achieved or something like that? 
thanks

**Comment by georgeef** [comment]
Maybe related: #6884, #6885. The idea is to archive (move) historical data to another file, which is not the same as keeping them in the same file and adding a flag to make them invisible.

**Comment by proleadsoft-inc** [comment]
f you're going through a divorce and need strong legal representation, I highly recommend this [divorce attorney in San Mateo](https://www.laughlinlegal.com/divorce/). They offer practical guidance and personalized support.

**Comment by Albert634515** [comment]
Maybe also related to: https://github.com/moneymanagerex/moneymanagerex/issues/6576
